assets/src/edit-story/components/checklist/priorityChecks.js

```diff
@@ -40,6 +47,12 @@ export function PriorityChecks() {
         <StoryPosterAspectRatio />
         <PublisherLogoSize />
         <VideoElementMissingPoster />
```

Comment:
There should also be check if the user can upload images here?

Reply:
See https://github.com/google/web-stories-wp/blob/6972495f63e209e32d157ba6a9c283add1ec4eb5/assets/src/edit-story/components/mediaPicker/useMediaPicker.js#L88-L95

Reply:
yeah we'll go through and do the permissions-related changes to the PPC in another PR. I handled it for the card I'm adding for this PR but we should go through all the checks to check for this permission.